tests/unit/models/project-test.js

```diff
@@ -465,11 +465,6 @@ describe('models/project.js', function() {
       var addon = project.findAddonByName('foo-ba');
       expect(addon).to.equal(undefined, 'foo-ba should not be found');
     })
-
-    it('should return an addon that is the unqualified name of requested name', function() {
```

Comment:
The goal is to only run the default blueprint if we can successfully parse a name out of the specified string using `node-package-args`.

We'll still run the `npm install` for things we don't recognize, but throw a console warning you that, "hey, we don't know your addon name, so you'll need to manually invoke your addon's generator blueprint."